packages/apps/kadena-docs/tsconfig.json

```diff
@@ -1,14 +1,9 @@
 {
   "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
   "compilerOptions": {
-    "types": [
-      "heft-jest",
-      "node"
-    ],
+    "types": ["heft-jest", "node", "@types/gtag.js"],
```

Comment:
`@types/gtag.js` shouldn't be needed as all `@types/*` is included by typescript in the `typeRoots` compiler option